.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
```

Comment:
Perhaps this addition requires a changelog entry to provide more information to other developers, like link to some popular IDE support of EditorConfig.

Reply:
Ah, good point. I'll update the changelog.